handbook/company/open-positions.yml

```diff
@@ -36,3 +36,26 @@
       - 🛠️ Technical: You understand the software development processes. You understand that software quality matters.
       - 🟣 Openness: You are flexible and open to new ideas and ways of working.
       - ➕ Bonus: Cybersecurity or IT background.
+- jobTitle: 🐋 Solutions Consultant
```

Comment:
FYI @dherder I think the line position here is what's causing this PR to fail the build.